homeassistant/components/wallbox/__init__.py

```diff
@@ -92,7 +93,10 @@ def __init__(self, station: str, wallbox: Wallbox, hass: HomeAssistant) -> None:
     def _authenticate(self) -> None:
```

Comment:
When is authenticate called ? 

The comment makes it sound it's emptied when it's about to expire, but the code just always empties it.

Can we put this logic in authenticate instead? It should refresh the token if it's too close. Alternative is to at least just pass a boolean to force refresh to `authenticate`. Like this it feels a bit weird

Reply:
Hi, yes, the comment was from the slightly more refined version. I now empty it all the time indeed. Basically the TTL given for the token seems buggy at best (refreshing at that time works sometimes, and than it doesn't, been struggling with it all weekend). I will update the comment.

Reply:
authenticate is called on updating data. As mentioned, I don't think this should be the endfix, but I'm struggling getting it to work. I'm buying time here, need to get to work tomorrow and currently the integration doesn't work at all.

Reply:
I addressed your comment, I have removed the clearing of the token. The issue is now resolved by a fix upstream (wallbox pypi package) has a fix in its token TTL.